src/voicefocus/VoiceFocusDeviceTransformer.ts

```diff
@@ -133,8 +133,28 @@ export class VoiceFocusDeviceTransformer {
   static async create(
     spec: VoiceFocusSpec = {},
     options: VoiceFocusDeviceOptions = {},
-    config?: VoiceFocusConfig
+    config?: VoiceFocusConfig,
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    createMeetingResponse?: any,
+    // eslint-disable-next-line
+    createAttendeeResponse?: any
   ): Promise<VoiceFocusDeviceTransformer> {
+    if (createMeetingResponse) {
```

Comment:
Can we just pass in `createMeetingResponse?.Meeting?.MeetingFeatures?.Audio?.EchoReduction === 'AVAILABLE';` as bool | undefined instead of the whole response?